InventoryComponents/InventoryRelicComponent.gd

```diff
@@ -0,0 +1,15 @@
+extends Node
+class_name InventoryRelicComponent
+
+var held_relics : Array[Relic]
+
+signal held_relics_update(new_relics : Array[Relic])
+
+func get_relic(relic : Relic) -> void:
```

Comment:
`add_relic` ? as I would expect `get` to return a relic instead of adding it to an array and returning nothing